### DISCUSSION TITLE
How to use spack-public mirror?

### DISCUSSION THREAD

**Discussion by jrobbin** [spack/spack#23261] [Q&A]
Do I need to tell spack to use the spack-public mirror?  I see it's listed ('spack mirror list'), but when I install something, say 'spack install xz', spack still uses the source tarball on sourceforge. 

Thanks!

**Comment by scheibelp** [comment]
Hi @jrobbin I think we spoke in Slack and I think the issue in your case was a problem with the SSL certs on the machine. I think the issue was that the download from them mirror failed, and in that case Spack was also trying to download from the original source (which also failed, but sometimes that would succeed if the only issue was the mirror being down). 

Generally the public mirror should be automatically available. This can be confirmed by doing `spack config blame mirrors` (the output should list a a "Spack public mirror".
